plugins/tiddlywiki/filesystem/filesystemadaptor.js

```diff
@@ -29,6 +29,70 @@ FileSystemAdaptor.prototype.name = "filesystem";
 
 FileSystemAdaptor.prototype.supportsLazyLoading = false;
 
+/*
+Show a filesystem errorlog alert
+*/
+FileSystemAdaptor.prototype.displayError = function(msg,err) {
+	// Check if there is an existing alert with the same title
+	var self = this,
+		alertTitle = "_alerts/filesytem/error",
+		logTitle = "_logs/filesytem/error",
+		alertTid = this.wiki.getTiddler(alertTitle),
+		logTid = this.wiki.getTiddler(logTitle),
+		alertFields = alertTid? $tw.utils.extend({},alertTid.fields): null,
+		logFields = logTid? $tw.utils.extend({},logTid.fields): null,
+		existingCount,
+		text = "<span style='float:left;margin:2pt 4pt 8pt 4pt'>{{$:/core/images/warning}}</span> " + msg + "; " + err.toString();
+	this.logger.alert(Array.prototype.join.call([
+		$tw.language.getString("Error/WhileSaving") + ":",
+		msg + ":",
+		err.toString()
+	]," "));
```

Comment:
IMO if the whole thing would be in 1 line it would be more readable. Especially, since the very last indent is wrong.

Reply:
IMO this is valid for the whole function... Not 100% sure, but IMO it would be worth a try, if it is more readable